README.md

```diff
@@ -117,6 +117,14 @@ and `api secret`.
 Trusona trusona = new Trusona(myApiToken, myApiSecret);
 ```
 
+The Trusona SDKs have infrastructure in place to serve the US, European and Asia-Pacific regions. By default, all SDK calls will be served by the US region. If the credentials you were provided by Trusona are specific to one of the other regions, then you’ll need to specify one of the possible values from the `Region` enum when building a `Trusona` instance.
```

Comment:
I think we should standardize on the few sentences in the JavaScript SDK.

> All users are provisioned in the default region. Unless otherwise noted you will not need to configure Trusona to use a specific region. If you have been provisioned in a specific region, you will need to point the SDK to use that region. 

Then get into the "how":

> If the credentials you were provided by Trusona are specific to one of the other regions, then you’ll need to specify one of the possible values from the `Region` enum when building a `Trusona` instance.

Reply:
I think just starting right away with `All users are provisioned in the default region. ` might not provide context as to which is the default region, so how about something like:

```
The Trusona SDKs have infrastructure in place to serve the US, European and Asia-Pacific regions, with the US being the default one. ```

Followed by:
```
Unless otherwise noted you will not need to configure Trusona to use a specific region.
```

I also think there would be some redundancy in the 
`If you have been provisioned in a specific region...` sentence with the 
`If the credentials you were provided by Trusona are specific to one of the other regions`

So maybe something like this would read better:

```
 If you have been provisioned in a specific region, you will need to point the SDK to use that region. To achieve this you’ll need to specify one of the possible values from the `Region` enum when building a `Trusona` instance.
```

So all together it would read like this:

```
The Trusona SDKs have infrastructure in place to serve the US, European and Asia-Pacific regions, with the US being the default one. Unless otherwise noted you will not need to configure Trusona to use a specific region.

If you have been provisioned in a specific region, you will need to point the SDK to use that region. To achieve this you’ll need to specify one of the possible values from the `Region` enum when building a `Trusona` instance.
```